ONSdigital/sdc-int-rh-ui
# Socint 264 no case

# Motivation and Context
Change the logic for the request new code address (uprn) not matching any case

# What has changed
Changes the faulty logic in the fulfilment journey 'Confirm address' screens to display the address and ask for confirmation before declaring there is no matching case. This allows the user to be sure of what they selected before being booted out of the process.
Change reinstates some of the Census code, so that if a matching case exists for the address, the address from the case is displayed, otherwise it falls back to use the address info from AIMS on the confirm screen (even though we know that this address does not have a matching case).
If the address is confirmed, but does not match a case the user is shown a page that their address is not in a a survey currently, and to call the CC
Change reinstates the get_ai_uprn call to AIMS in the fallback.

# How to test?
Start request a new code journey
Select an address that does not have a case
Confirm selection
Check get appropriate screen
Also, check that picking an address that does have a case works as normal

# Links
https://collaborate2.ons.gov.uk/jira/browse/SOCINT-264
